File: backend/Api/LeagueSquadApi/Dtos/SquadMatchDtos.cs
namespace LeagueSquadApi.Dtos
{
    public record SquadMatchResponse(long SquadId, string MatchId, string? ReasonForAddition, int QueueId, DateTimeOffset GameStart, DateTimeOffset GameEnd, int DurationSeconds, string Mode, string GameType, int MapId, DateTimeOffset CreatedAt);
}

File: frontend/league-squad-web/src/types/SquadMatchDtos.ts
export interface SquadMatchResponse {
    squadId: number,
    matchId: string,
    reasonForAddition?: string,
    queueId: number,
    gameStart: string,
    gameEnd: string,
    durationSeconds: number,
    mode: string,
    gameType: string,
    mapId: number,
    createdAt: number
}

// export interface SquadMatchRequest {

// }